### ISSUE TITLE
ELM generation fails

### ISSUE THREAD

**Issue by aeyates** [OHSUCMP/htnu18ig#116]
Sometime last year, we started getting this error when refreshing for the Pharma, NonPharma, Common, and Monitoring CQL files:

CQL Translation succeeded for file: '/Users/yateam/HTN/htnu18ig/input/cql/OHSUHTNCommon.cql', but ELM generation failed with the following error: Expected ElmExpressionRequirement

This does not prevent us from packaging the CQL and pushing it to CQF Ruler, and our tests all still pass.

**Comment by aeyates** [comment]
I did a little digging on this. It is likely related to this bug reported by Bryn last year that is still open:

https://github.com/cqframework/cqf-tooling/issues/248

In the common library, the first definition that triggers the error is this:

`// Does patient have a BP goal?
define "Qualifying Blood Pressure Goals":
  (QualifiedGoal(["Goal"])) BPGoal
    where (
      singleton from (BPGoal.target Systolic
        where Systolic.measure in "Systolic blood pressure"
          and Systolic.detail is Quantity
          and WithUnit(Systolic.detail, 'mm[Hg]'))
    ) SystolicTarget is not null
      and (
        singleton from (BPGoal.target Diastolic
          where Diastolic.measure in "Diastolic Blood Pressure"
            and Diastolic.detail is Quantity
            and WithUnit(Diastolic.detail, 'mm[Hg]'))
      ) DiastolicTarget is not null
`

Basically any of these fields could be null since they are not required in FHIR. Also, a goal can have multiple targets which this doesn't account for. But I don't think it's worth rewriting since it works and the error being thrown has been identified as a bug in cqf tooling.
